recipe/build_hoomd.sh

```diff
@@ -27,7 +28,7 @@ cmake ../ \
       -DCMAKE_INSTALL_PREFIX=${SP_DIR} \
       -DPYTHON_EXECUTABLE=${PYTHON} \
       -DENABLE_MPI=off \
-      -DENABLE_CUDA=${CUDA_SUPPORT} ${CUDA_CMAKE_OPTIONS} \
+      -DENABLE_GPU=${CUDA_SUPPORT} ${CUDA_CMAKE_OPTIONS} \
```

Comment:
Should this be quoted?

```suggestion
      -DENABLE_GPU="${CUDA_SUPPORT} ${CUDA_CMAKE_OPTIONS}" \
```